x-pack/plugins/maps/common/descriptor_types/map_descriptor.ts

```diff
@@ -41,6 +42,7 @@ export type Goto = {
 export type TooltipFeature = {
   id: number;
```

Comment:
I think `id` typing needs to be updated based on how its being used. Should be `id? : string | number`

Reply:
ok. requires corresponding change in `LoadFeatureProps` as well.